CONTRIBUTING.md

```diff
@@ -0,0 +1,49 @@
+# Contributing to the AeroGear Android SDK
+
+The AeroGear Android SDK is part of the [AeroGear project](https://aerogear.org/), see the [Community Page](https://aerogear.org/community) for general guidelines for contributing to the project.
+
+This document details specifics for contributions to the Android SDK.
+
+## Issue tracker
+
+The tracking of issues for the AeroGear Android SDK is done in the [AeroGear Android Project](https://issues.jboss.org/projects/AGDROID/issues) in the [JBoss Developer JIRA](https://issues.jboss.org).
+
+See the [AeroGear JIRA Usage and Guidelines Guide](https://aerogear.org/docs/guides/JIRAUsage/) for information on how the issue tracker relates to contributions to this project.
+
+## Asking for help
+
+Whether you're contributing a new feature or bug fix, or simply submitting a
+ticket, the Aerogear team is available for technical advice or feedback. 
+You can reach us at [#aerogear](ircs://chat.freenode.net:6697/aerogear) on [Freenode IRC](https://freenode.net/) or the 
+[aerogear-dev list](http://lists.jboss.org/pipermail/aerogear-dev/)
+-- both are actively monitored.
+
+## Android SDK Overview and Architecture
```

Comment:
IMHO Contents bellow should not be in the contribution guide.
They will quickly get outdated anyway. 

This will be good to have in general documentation repo. 
Do you still remember: http://raincatcher.feedhenry.io/docs/#Contributing :P

Reply:
agreed. It's better to move this section to its own document. We can keep it in the same repo for now.

Reply:
Will do, there's already a `docs/` folder that @pb82 added the info for the BOM usage